RELEASE_NOTES.md

```diff
@@ -3,8 +3,38 @@ Release notes
 
 ### Next
 
-* Included support for compilation on Java 9 and 10. Compilation on Java 9 or 10 will produce a multi-release JAR that will run on Java 8, 9, and 10. This release was compiled on Java 8 and does not contain a multi-release JAR.
-* Changed metadata health check names to use the URI of the metadata they are trying to resolve
+#### Connect to Verify using only the VSP
+
+This release adds the ability to connect to GOV.UK Verify using only the Verify Service Provider (VSP). This means services can connect without needing to host a [Matching Service Adapter (MSA)](https://github.com/alphagov/verify-matching-service-adapter). If needed, services can then implement matching independently from their connection to GOV.UK Verify.
+
+Using only the VSP makes it easier to connect to GOV.UK Verify and reduces maintenance tasks once connected.
+
+When used alone, the VSP provides the required MSA functionality. It will:
+* implement security features to the same level as the MSA
+* handle both GOV.UK Verify and European identities
+* do Personal Identifier (PID) hashing
+
+If you are already connected to GOV.UK Verify and are running an MSA, you can upgrade to VSP 2.0.0 without needing to do any configuration changes.
+
+If you want to switch off your Matching Service Adapter and only use the Verify Service Provider to connect to Verify, [contact the GOV.UK Verify Team](https://www.verify.service.gov.uk/support/).
+
+For more information on connecting to GOV.UK Verify using only the VSP, see the [technical documentation](https://www.docs.verify.service.gov.uk).
+
+#### Improved technical documentation
+
+The [new technical documentation website](https://www.docs.verify.service.gov.uk) has new guidance to help users connect to GOV.UK Verify using the VSP.
```

Comment:
The _new_ technical ... has _new_ guidance ...
Would this read better with one of the 'new's omitted or changed up? Maybe switch the first one for 'improved'?

Reply:
I've rephrased :)